packages/lmi/src/lmi/types.py

```diff
@@ -76,6 +76,30 @@ class LLMResult(BaseModel):
     completion_count: int = 0
     model: str
     date: str = Field(default_factory=datetime.now().isoformat)
+
+    # Cached token counts - extracted from provider-specific usage fields:
+    # - Both providers report cache reads via prompt_tokens_details.cached_tokens
+    # - Only Anthropic reports cache creation via cache_creation_input_tokens
+    cache_read_tokens: int | None = Field(
+        default=None,
+        description=(
+            "Tokens read from cache (Anthropic/OpenAI). "
+            "None means caching wasn't used, 0 means caching was used but no cache hits."
+        ),
+    )
+    cache_creation_tokens: int | None = Field(
+        default=None,
+        description=(
+            "Tokens written to cache (Anthropic only). "
+            "None means caching wasn't used, 0 means caching was used but no cache creation."
+        ),
+    )
+
+    # Store accurate cost from litellm (handles all pricing tiers and provider quirks)
+    cost_usd: float | None = Field(
+        default=None,
+        description="Calculated by litellm.completion_cost(). Accurate for cached tokens.",
```

Comment:
```
    cost: float | None = Field(
        default=None,
        description="Cost (USD).",
```

A few nits:
- Don't put "Accurate for cache tokens" -- imo it's expected to be accurate for all kinds of tokens
- Don't put `litellm.completion_cost` -- that's an implementation detail that may change over time
- Don't put units in variable names (and feel free to ignore this one)